io.catenax.battery.battery_pass/3.0.1/BatteryPass.ttl

```diff
@@ -0,0 +1,920 @@
+:ratioMaximumAllowedBatteryPowerAndMaximumAllowedBatteryEnergy a bamm:Property;
+    bamm:name "ratioMaximumAllowedBatteryPowerAndMaximumAllowedBatteryEnergy";
+    bamm:preferredName "ratio maximum allowed battery power and maximum allowed battery energy"@en;
+    bamm:description "Ratio between maximum allowed battery power (W) and battery energy (Wh) is describing a regulatory requirement."@en;
+    bamm:characteristic :RatioMaximumAllowedBatteryPowerAndMaximumAllowedBatteryEnergyCharacterisitic;
+    bamm:exampleValue "0.611"^^xsd:float.
+:length a bamm:Property;
+    bamm:name "length";
+    bamm:preferredName "length"@en;
+    bamm:description "Length of the item is describing a business requirement."@en;
+    bamm:characteristic :MillimetreCharacteristic;
+    bamm:exampleValue "20000"^^xsd:integer.
+:width a bamm:Property;
+    bamm:name "width";
+    bamm:preferredName "width"@en;
+    bamm:description "Width of the item is describing a business requirement."@en;
+    bamm:characteristic :MillimetreCharacteristic;
+    bamm:exampleValue "1000"^^xsd:integer.
+:height a bamm:Property;
+    bamm:name "height";
+    bamm:preferredName "height"@en;
+    bamm:description "Height of the item is describing a business requirement."@en;
+    bamm:characteristic :MillimetreCharacteristic;
+    bamm:exampleValue "1"^^xsd:integer.
+:diameter a bamm:Property;
+    bamm:name "diameter";
+    bamm:preferredName "diameter"@en;
+    bamm:description "Diameter of the item."@en;
+    bamm:characteristic :MillimetreCharacteristic;
+    bamm:exampleValue "3"^^xsd:integer.
+:weight a bamm:Property;
+    bamm:name "weight";
+    bamm:preferredName "weight"@en;
+    bamm:description "Weight of the item is describing a regulatory requirement."@en;
+    bamm:characteristic :KilogramCharacteristic;
+    bamm:exampleValue "1007"^^xsd:integer.
+:stateOfHealth a bamm:Property;
+    bamm:name "stateOfHealth";
+    bamm:preferredName "state of health"@en;
+    bamm:description "Evidence/Certificate of the health evaluation of a battery for its use following repurposing or remanufacturing operations is describing a business requirement."@en;
+    bamm:characteristic :StateOfHealthCharacteristic;
+    bamm:exampleValue "12"^^xsd:integer.
+:stateOfCharge a bamm:Property;
+    bamm:name "stateOfCharge";
+    bamm:preferredName "state of charge"@en;
+    bamm:description "The value of the state of charge of the battery at the end of life i.e. when entering recycling scheme is describing a business requirement."@en;
+    bamm:characteristic :StateOfChargeCharacteristic;
+    bamm:exampleValue "23"^^xsd:integer.
+:statusBattery a bamm:Property;
+    bamm:name "statusBattery";
+    bamm:preferredName "status battery"@en;
+    bamm:description "Status of the battery is describing a regulatory requirement. Value list provided by the regulators."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "first life/ waste/ repaired/ repurposed/ recycled".
```

Comment:
IF there is a fixed set of values wouldn't be using an enum be more appropriate?

Reply:
no fixed set, just possible examples